src/App.jsx

```diff
@@ -62,57 +230,57 @@ const RPGInterface = () => {
         </div>
 
         {/* Health Bars */}
-        {!menu && (
+        {!(gameState.gameScene === GameScene.MENU_SCENE) && (
```

Comment:
This was originally my bad for an unnecessary `!`. Here we should change this line:
```suggestion
        {gameState.gameScene === GameScene.BATTLE_SCENE) && (
```

Reply:
Oops, I left in the right parenthesis.